dts/bindings/dma/arm,dma-pl330.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Broadcom
+# SPDX-License-Identifier: Apache-2.0
+
+description: PL330 DMA Controller
+
+compatible: "arm,dma-pl330"
+
+include: dma-controller.yaml
+
+properties:
+    reg:
+      required: true
+    microcode:
+      type: array
+      required: true
+    label:
+      required: true
+
+
+description: |
```

Comment:
this seems odd, why description at line 20 and line 4.

Reply:
ok, let me fix it. Ack